snowplow-demo-new/src/main/java/com/snowplowanalytics/snowplow_demo_new/ui/SchemaDetailScreen.kt

```diff
@@ -0,0 +1,146 @@
+package com.snowplowanalytics.snowplow_demo_new.ui
+
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.rememberScrollState
+import androidx.compose.foundation.verticalScroll
+import androidx.compose.material.*
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.filled.ArrowBack
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.LaunchedEffect
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.TextStyle
+import androidx.compose.ui.text.style.TextOverflow
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import com.snowplowanalytics.snowplow.event.ScreenView
+import com.snowplowanalytics.snowplow.payload.SelfDescribingJson
+import com.snowplowanalytics.snowplow_demo_new.R
+import com.snowplowanalytics.snowplow_demo_new.data.Tracking
+import java.util.*
+
+
+@Composable
+fun SchemaDetailScreen(
+    vm: SchemaDetailViewModel,
+    schemaUrl: String,
+    onBackButtonClicked: () -> Unit = {}
+) {
+    LaunchedEffect(Unit, block = {
+        vm.getSchemaDescription(schemaUrl)
+        vm.getSchemaJson(schemaUrl)
+    })
+    val schemaParts = vm.processSchemaUrl(schemaUrl)
+    
+    val entity = SelfDescribingJson(
+        "iglu:com.snowplowanalytics.iglu/anything-a/jsonschema/1-0-0", 
+        hashMapOf("name" to schemaParts.name, "vendor" to schemaParts.vendor)
+    )
+    val event = ScreenView("detail", UUID.randomUUID()).entities(listOf(entity))
+
+    // Tracks a ScreenView with attached context entity.
+    // This entity records information about the specific schema being viewed
+    Tracking.tracker()?.track(event)
```

Comment:
Could we use a `DisposableEffect` as [suggested here](https://stackoverflow.com/a/69464673) to track the screen view? Instead, I think this will track the event on every re-render of the component which might be more often than every screen.